### ISSUE TITLE
[Suggestion] Add the Integrated Tunnels mod

### ISSUE THREAD

**Issue by Delerium** [Darkosto/SevTech-Ages#2274]
Right now using integrated dynamics to automate is extremely limiting with our transport options.  Mekanism transporters don't respond to redstone, even diversion ones that specifically state that they do,  Hoppers do respond to redstone but still provides rather limited integration with ID.  It can still be gated to stage 4 like the rest of ID.

**Comment by andybond006** [comment]
Now that JustEnoughIDs has been added, does this mod have any chance to be added in coming updates? I think the addition of Integrated Tunnels would add a lot of incentive for more players to learn the Integrated suite of mods as they are underutilized in most modpacks that contain Thermal and EnderIO.

**Comment by tommysundling** [comment]
I agree with @andybond006 in that the Integrated mods doesn't appear to be used extensively by players in other modpacks (based on my personal YouTube viewing) so I think it adding Integrated Tunnels to SevTech would fit in nicely with the different style of doing things.

If it's considered too powerful then gate it behind Age 5.

**Comment by rtsking117** [comment]
Integrated Tunnels practically integrates itself - only the fluid/energy/item importer/exporter/interface devices are craftable in stage 4 as everything else requires materials already gated to stage 5 (logic directors/emeralds). Most importantly (in my opinion) the player simulator, world block importer, and world block exporter (auto clicker, miner, and placer) require both diamonds and emeralds, meaning you must make it to stage 5 before they can be used at all. 

There may need to be a bit of .zs file scripting to adequately lock off the recipes until age 4, but since it doesn't look like that's done even with Integrated Dynamics it may "just work" in the end.

**Comment by sam-kirby** [comment]
As the entirety of ID is staged to 4, it is done by mod id here: https://github.com/DarkPacks/SevTech-Ages/blob/master/src/scripts/crafttweaker/staging/itemsAndRecipes/modId.zs#L35

Having items from the same mod in different stages does mean doing it by item. Not doing so will leave the items visible to a new player.
